tests/ipsecondary_test.py

```diff
@@ -201,8 +229,7 @@ def test_primary_timeout_after_device_is_registered(uptane_repo, secondary, aktu
     with aktualizr:
         aktualizr.wait_for_completion()
 
-    return (aktualizr.get_current_primary_image_info() == primary_image_hash)\
-            and not aktualizr.is_ecu_registered(secondary.id)
+    return aktualizr.get_current_primary_image_info() == primary_image_hash
```

Comment:
What necessitated simplifying this condition?

Reply:
In this PR, even if a secondary is not connected when aktualizr starts, it is still considered as present.

`is_ecu_registered()` only returns True if the primary abandoned the idea of running the secondary after the original time out and would not send updates its way.